go.mod

```diff
@@ -8,31 +8,31 @@ require (
 	github.com/gofrs/flock v0.7.1
 	github.com/google/go-cmp v0.5.6
 	github.com/google/go-tpm v0.3.2
-	github.com/prometheus/client_golang v1.11.0
-	github.com/spf13/cobra v1.2.1
+	github.com/prometheus/client_golang v1.12.1
```

Comment:
This has no effect due to line 147 replacing this with v1.7.1, can you please update the replace statement?

Reply:
Yeah, thanks!